utils/debug/Debug.js

```diff
@@ -95,6 +113,80 @@ function Debug(view, datDebugTool, chartDivContainer) {
         // debug graphs update
         debugChartUpdate(duration);
     });
+
+    // Camera debug
+    const helper = new CameraHelper(view.camera.camera3D);
+    const debugCamera = view.camera.camera3D.clone();
+    debugCamera.fov *= 1.5;
+    debugCamera.updateProjectionMatrix();
+    const g = view.mainLoop.gfxEngine;
+    const r = g.renderer;
+    let fogDistance = view.fogDistance;
+    helper.visible = false;
+    view.scene.add(helper);
+
+    function tileDisplay(obj) {
```

Comment:
It looks like the name of the function doesn't really reflect its actions. Can you find a more descriptive name?

Reply:
renamed